tomoScanApp/op/adl/tomoScan.adl

```diff
@@ -1,14 +1,14 @@
 
 file {
```

Comment:
I downloaded and viewed the new version of tomoScan.adl in medm.  It has a number of problems:
- The total height of the display is wrong, the bottom is cut off.  It was previously 825 but was changed to 803 for some reason.
- All widget X and Y positions should be a multiple of 5.  That keeps them correctly aligned. 
- Text labels should be positioned immediately adjacent to the widget they describe using "Space Evenly/Horizontal"
- Text labels should be exactly the size required by the text using "Size/Text to Contents".